src/screen/credits.rs

```diff
@@ -12,9 +12,12 @@ pub(super) fn plugin(app: &mut App) {
         Update,
         handle_credits_action.run_if(in_state(Screen::Credits)),
     );
+
+    app.register_type::<CreditsAction>();
```

Comment:
Group this up with the line above, immediately before it.

Reply:
[`e09bbe5` (#78)](https://github.com/TheBevyFlock/bevy-template/pull/78/commits/e09bbe52f1c9afa85b24d54d47e8e0f4570b8ebe)